libopenage/gamestate/team.cpp

```diff
@@ -61,4 +64,12 @@ member_type Team::get_member_type(Player &player) {
 	return member_type::none;
 }
 
+std::vector<const Player*> Team::get_players() const {
```

Comment:
`warning: reference to local variable`

It will be optimized into a pointer pass anyway, I was talking more about the vector construction adding one by one the players. But its ok, it's a small list...